x-pack/legacy/plugins/infra/public/utils/formatters/datetime.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export function localizedDate(dateTime: number | Date, locale?: string) {
```

Comment:
Can apply the defaults here instead of checking/updating in the function body
```suggestion
export function localizedDate(dateTime: number | Date, locale: string = i18n.getLocale()) {
```